### ISSUE TITLE
Spack's go crashes

### ISSUE THREAD

**Issue by matthewsucar** [NCAR/spack-gust#26]
It'd be nice if this didn't happen:

```matthews@gust01:~> ml go
matthews@gust01:~> go version
Segmentation fault (core dumped)
matthews@gust01:~> go run test.go
Segmentation fault (core dumped)
matthews@gust01:~> which go
/glade/u/apps/gust/22.08b/spack/opt/spack/go/1.18/gcc/7.5.0/bin/go
```

**Comment by benkirk** [comment]
Should be resolved as of 22.10:

```pre
$ module purge
$ module load ncarenv/22.10 
$ module load go
$ go --version
flag provided but not defined: -version
Go is a tool for managing Go source code.

Usage:

	go <command> [arguments]

...
```

**Comment by vanderwb** [comment]
Should work as documented by @benkirk. Closing.
